travis failed on ProcessingToolTest with something not related with this PR but with OGR tests
test_ogrLayerName

======================================================================
FAIL: test_ogrLayerName (__main__.VectorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/qgis/QGIS/python/plugins/processing/tests/ToolsTest.py", line 81, in test_ogrLayerName
    self.assertEqual(name, 'a')
AssertionError: 'b' != 'a'

--- a/src/core/symbology-ng/qgsellipsesymbollayerv2.cpp
+++ b/src/core/symbology-ng/qgsellipsesymbollayerv2.cpp
@@ -490,7 +490,7 @@ QgsSymbolLayerV2* QgsEllipseSymbolLayerV2::createFromSld( QDomElement &element )
   if ( !QgsSymbolLayerV2Utils::wellKnownMarkerFromSld( graphicElem, name, fillColor, borderColor, borderStyle, borderWidth, size ) )
     return nullptr;
 
-  QString uom = element.attribute( QString( "uom" ), "" );
+  const QString uom = element.attribute( QString( "uom" ), "" );
   size = QgsSymbolLayerV2Utils::sizeInPixelsFromSldUom( uom, size );
   borderWidth = QgsSymbolLayerV2Utils::sizeInPixelsFromSldUom( uom, borderWidth );
 

--- a/tests/src/python/test_qgssymbollayerv2_readsld.py
+++ b/tests/src/python/test_qgssymbollayerv2_readsld.py
@@ -331,13 +331,8 @@ class TestQgsSymbolLayerReadSld(unittest.TestCase):
 
         sld_size_px = 6
 
-        # values for qgis 3.X
-        #sld_displacement_x_px = 3.3
-        #sld_displacement_y_px = 6.6
-
-        # values for qgis 2.x
-        sld_displacement_x_px = 0
-        sld_displacement_y_px = 0
+        sld_displacement_x_px = 3.3
+        sld_displacement_y_px = 6.6
 
         sl = layer.rendererV2().symbol().symbolLayers()[0]
         size = sl.size()

--- a/tests/src/python/test_qgssymbollayerv2_readsld.py
+++ b/tests/src/python/test_qgssymbollayerv2_readsld.py
@@ -151,7 +151,7 @@ class TestQgsSymbolLayerReadSld(unittest.TestCase):
         # create a layer
         layer = createLayerWithOnePoint()
 
-         # load a sld with marker size without uom attribute (pixels)
+        # load a sld with marker size without uom attribute (pixels)
         sld = 'symbol_layer/QgsSvgMarkerSymbolLayerV2.sld'
         mFilePath = os.path.join(TEST_DATA_DIR, sld)
         layer.loadSldStyle(mFilePath)

--- a/tests/testdata/symbol_layer/QgsSimpleMarkerSymbolLayerV2.sld
+++ b/tests/testdata/symbol_layer/QgsSimpleMarkerSymbolLayerV2.sld
@@ -9,6 +9,10 @@
           <se:Name>Single symbol</se:Name>
           <se:PointSymbolizer>
             <se:Graphic>
+              <se:Displacement>
+                <se:DisplacementX>3.3</se:DisplacementX>
+                <se:DisplacementY>6.6</se:DisplacementY>
+              </se:Displacement>
               <se:Mark>
                 <se:WellKnownName>pentagon</se:WellKnownName>
                 <se:Fill>

